.buildkite/docker-compose.yml

```
@@ -37,6 +37,8 @@ services:
       - BUILDKITE_COMMIT
       - BUILDKITE_TAG
       - CODECOV_TOKEN
+      - DOCKER_CLIENT_TIMEOUT=360
```

If we were to land this, it would be good to have a comment here and in the other one:

```suggestion
      # the images are large, so pushing with the defaults can timeout
      - DOCKER_CLIENT_TIMEOUT=360
```